imgix/urlbuilder.py

```diff
@@ -100,21 +107,76 @@ def create_url(self, path, params=None):
         str
             imgix URL
         """
-        if not params:
-            params = {}
+        sanitized_path = self._sanitize_path(path)
 
-        domain = self._domain
-        scheme = "https" if self._use_https else "http"
+        query_string = self._build_params(params)
+        if self._sign_key:
+            query_string = self._sign_url(sanitized_path, query_string)
 
-        url_obj = UrlHelper(
-            domain,
-            path,
-            scheme,
-            sign_key=self._sign_key,
-            include_library_param=self._include_library_param,
-            params=params)
+        scheme = "https" if self._use_https else "http"
 
-        return str(url_obj)
+        return scheme + "://" \
+            + self._domain \
+            + sanitized_path \
+            + query_string
+
+    def _sanitize_path(self, path):
+        if not path:
+            return ""
```

Comment:
qq: should there be some kind of error thrown? Maybe a warning log? Idk when `path` would eval to `false` here other than on invalid input.

Reply:
Maybe, I've thought of adding a `and not isinstance(path, str)` clause here